tokio/src/net/tcp/stream.rs

```diff
@@ -184,6 +184,56 @@ impl TcpStream {
         Ok(TcpStream { io })
     }
 
+    /// Turn a [`tokio::net::TcpStream`] into a [`std::net::TcpStream`].
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::error::Error;
+    /// use std::io::Read;
+    /// use tokio::net::TcpListener;
+    /// # use tokio::net::TcpStream;
+    /// # use tokio::io::AsyncWriteExt;
+    /// # use tokio::time::Duration;
+    ///
+    /// #[tokio::main]
+    /// async fn main() -> Result<(), Box<dyn Error>> {
+    /// #   tokio::spawn(async {
+    /// #       tokio::time::sleep(Duration::from_millis(50)).await;
+    /// #       let mut stream: TcpStream = TcpStream::connect("127.0.0.1:8080").await.unwrap();
+    /// #       let _ = stream.write(b"Hello world!").await;
```

Comment:
```suggestion
    /// #       stream.write(b"Hello world!").await.unwrap();
```

Reply:
Seems I need this otherwise I see this during test?
```
running 1 test
test src/net/tcp/stream.rs - net::tcp::stream::TcpStream::into_std (line 191) ... FAILED

failures:

---- src/net/tcp/stream.rs - net::tcp::stream::TcpStream::into_std (line 191) stdout ----
error: unused `std::result::Result` that must be used
  --> src/net/tcp/stream.rs:204:7
   |
15 |       stream.write(b"Hello world!").await;
   |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
You need the `unwrap()` too.

Reply:
Aha, thanks 😄